unstract/connectors/src/unstract/connectors/databases/oracle-db/static/json_schema.json

```diff
@@ -19,33 +19,39 @@
     },
     "config_dir": {
       "type": "string",
-      "title": "Specifies the directory containing tnsnames.ora ",
+      "title": "Connector Directory",
+      "description": "Specifies the directory containing tnsnames.ora ",
       "default": "/opt/OracleCloud/MYDB"
     },
     "user": {
       "type": "string",
-      "title": "Specifies the database user ",
+      "title": "User",
+      "description": "Specifies the database user ",
       "default": "admin"
     },
     "password": {
       "type": "string",
-      "title": "Specifies the database user password ",
+      "title": "Password",
+      "description": "Specifies the database user password ",
       "format": "password",
       "default": ""
     },
     "dsn": {
       "type": "string",
-      "title": "Specify the desired network or the TNS names",
+      "title": "Dsn",
+      "description": "Specify the desired network or the TNS names",
       "default": "db2024_low"
```

Comment:
Does this default apply to most cases? Can you link some documentation on this? If its not every common, we should leave it empty instead maybe